Realm/TestUtils/include/RLMTestObjects.h

```diff
@@ -21,9 +21,11 @@
 #if __has_extension(objc_generics)
 #define RLM_GENERIC_ARRAY(CLASS) RLMArray<CLASS *><CLASS>
 #define RLM_GENERIC_SET(CLASS) RLMSet<CLASS *><CLASS>
+#define RLM_GENERIC_DICTIONARY(KEY, VALUE) RLMDictionary<KEY, VALUE>
```

Comment:
```suggestion
#define RLM_GENERIC_DICTIONARY(KEY, VALUE) RLMDictionary<KEY *, VALUE *><KEY, VALUE>
```

The purpose of these macros is to define the properties with generic types without repeating the names.

Reply:
@tgoyne , can we talk about it as I see it works fine?